rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/BookKeeper.java

```diff
@@ -542,7 +542,7 @@ private boolean readDataInternal(String remotePath, long offset, int length, lon
           fs.close();
         }
         catch (IOException e) {
-          e.printStackTrace();
+          log.error(String.format(Throwables.getStackTraceAsString(e)));
```

Comment:
Remove `String.format()` on this and similar lines; this should only be used for formatting strings (ie. strings containing `%s` and the like)